dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/offset/PeriodOffsetUtils.java

```diff
@@ -0,0 +1,152 @@
+package org.hisp.dhis.analytics.offset;
```

Comment:
Move to package `org.hisp.dhis.analytics.utils`.

Reply:
Fix